README.md

```diff
@@ -15,7 +15,17 @@ _Click on the GIF below to see a full example run_
 __Documentation is available at [https://docs.luxonis.com/en/latest/pages/tutorials/first_steps/](https://docs.luxonis.com/en/latest/pages/tutorials/first_steps/#first-steps-with-depthai).__
 
 ## Installation
-
+First you need to clone this repository with
+```
+git clone https://github.com/luxonis/depthai.git
+cd depthai
+git pull
+git submodule update --init
```

Comment:
`git clone --recursive https://github.com/luxonis/depthai.git`

Reply:
Change made in new commit.